src/udev/udev-builtin-net_id.c

```diff
@@ -243,9 +243,29 @@ static bool is_pci_ari_enabled(sd_device *dev) {
         return streq(a, "1");
 }
 
+static bool is_pci_bridge(sd_device *dev) {
+        const char *v, *p;
+
+        if (sd_device_get_sysattr_value(dev, "modalias", &v) < 0)
+                return false;
+
+        if (!startswith(v, "pci:"))
+                return false;
+
+        p = strrchr(v, 's');
+        if (!p)
+                return false;
+        if (*++p != 'c')
```

Comment:
i must admit, when i see expressions such as `*++p` i have to think twice what this means. I'd really write this as p[1] and then call strneq() below with `p + 2` as string.

Reply:
Confusing and hard to read code is just udev's coding style, trying to stay consistent here ;)

Sure, I will rewrite it.